packages/cross_file/pubspec.yaml

```diff
@@ -1,8 +1,13 @@
 name: cross_file
 description: An abstraction to allow working with files across multiple platforms.
 repository: https://github.com/flutter/packages/tree/master/packages/cross_file
+issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+animations%22
```

Comment:
```suggestion
issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+cross_file%22
```

Reply:
Good catch, I thought I did a final sweep for copypasta mistakes.